src/screens/Customers/components/CustomerCard.tsx

```diff
@@ -0,0 +1,139 @@
+      color: theme.palette.text.secondary,
+    },
+    numberText: {
+      color: theme.palette.error.main,
+      marginLeft: '5px',
+    },
+    divSpacing: {
+      paddingLeft: '15px',
+    },
+    singleCard: {
+      width: '100%',
+      height: '93px',
+      border: '1px solid ' + theme.palette.text.disabled,
+      borderRadius: '8px',
+      textAlign: 'left',
+      display: 'flex',
+      flexDirection: 'row',
+      marginBottom: '10px',
+    },
+    active: {
+      height: '7px',
+      width: '7px',
+      borderRadius: '50%',
+      backgroundColor: theme.palette.secondary.light,
+    },
+    notActive: {
+      height: '7px',
+      width: '7px',
+      borderRadius: '50%',
+      backgroundColor: theme.palette.text.disabled,
+    },
+    arrowSpacing: {
+      padding: '0px',
+    },
+    indicator: {
+      display: 'flex',
+      alignItems: 'center',
+      marginRight: '10px',
+    },
+    icon: {
+      borderRadius: '8px',
+      color: 'white',
+      padding: '5px',
+      backgroundColor: theme.palette.primary.main,
+      fontSize: '30px',
+    },
+  });
+
+interface CardProps {
+  name: string;
+  customerId: string;
+  meterNumber: number;
+  amount: string;
+  active: boolean;
+  match: any;
+  status?: CustomerStatus;
+  classes: {
+    cardContent: string; singleCard: string; totalText: string; numberText: string; divSpacing: string; active: string; notActive: string; arrowSpacing: string; indicator: string; icon: string;
+  };
+}
+
+export enum CustomerStatus {
+  METER = 'Meter' as any,
+  PAYMENT = 'Payment' as any,
+  DONE = 'Done' as any,
+}
+
+function CustomerCard(props: CardProps) {
+  const { classes } = props;
+
+  const getStatusIcon = () => {
+    /* TODO: replace paths /customer/edit with actual paths */
+    if (props.status === CustomerStatus.METER) {
+      return (
+        <IconButton
+          component={Link}
+          to={{
+            pathname: `${props.match.url}/customer/edit`,
+          }}
+          size="small"
+        >
+          <FlashOnIcon className={classes.icon} fontSize='inherit' />
+        </IconButton>
+      );
+    } else if (props.status === CustomerStatus.PAYMENT) {
+      return (
+        <IconButton
+          component={Link}
+          to={{
+            pathname: `${props.match.url}/customer/edit`,
+          }}
+          size="small"
+        >
+          <AttachMoneyIcon className={classes.icon} fontSize='inherit' />
+        </IconButton>
+      );
+    }
+  }
+
+  return (
+      <Card className={classes.singleCard} variant="outlined">
+        <Link to={`${props.match.url}/customer`} onClick={() => setCurrentCustomerIdInRedux(props.customerId)} className={classes.cardContent}>
+          <div className={classes.indicator}>
+            {props.active ? <div className={classes.active} /> : <div className={classes.notActive} />}
+          </div>
+          <div>
+            <Typography variant="h2">{props.meterNumber}, {props.name}</Typography>
```

Comment:
This should actually be the `customerNumber` not `meterNumber`

Reply:
should I do some error handling or should I assume that everyone has a customerNumber?

Reply:
Something basic would be good just so the app doesn't error, but this should be a required field and all records should have it. I jsut changed the Airtable records so all Customers have a unique customerNumber